wordpress-mobile/WordPress-Aztec-iOS
# Use custom paragraph style for lists.

So I decided to change our approach for lists, instead of using a custom attribute I implemented a new ParagraphStyle class that inherits from NSMutableParagraphStyle, that implements the textList attribute in a similar way macOS does.

The big advantages of this is that we get all the automatic magic of copying attributes by the default implementation  of TextView so we don't need to applied themselves when changes happen in the list.

How to test:
 - Launch the demo app
 - Check if the initial import from HTML goes correctly
 - Play around with lists by toggling them, adding new lines, delete lines and see if all works well.


Note: This still doesn't implement the updates on the DOM so any changes done on the string after the initial import from HTML are not reflected back to the HTML
